src/data_operations.py

```
@@ -0,0 +1,100 @@
+        if not data_type.startswith('SOFR'):
+            data_df = fetch_and_process_data(fred, data_info, col_date)
+            save_data(data_df, data_type, dataPath, SAVE_AS_PICKLE)
+            audit_data.append(collect_audit_info(data_df, data_type, data_info['data_ref']))
+            if PUSH_TO_GCP:
+                upload_to_gcp(data_df, data_type, bucket, SAVE_AS_PICKLE)
+            print(f"# {data_type}: Updated")
+            time.sleep(1)
+    return audit_data
+
+
+def process_sofr_data(sofr_series, fred, col_date, dataPath, SAVE_AS_PICKLE, PUSH_TO_GCP, bucket):
```

Ruff (0.8.2)

Missing return type annotation for public function `process_sofr_data`

(ANN201)